api/config.py

```diff
@@ -262,11 +280,14 @@ def get_config():
     return __config
 
 def get_public_config():
+    auth = copy.deepcopy(__config.get('auth'))
+    for values in auth.itervalues():
+        values.pop('client_secret', None)
```

Comment:
Yeah, up to you.
Personally, I would rather a whitelist be long and incomplete than a blacklist be incomplete.